integration/init_app/integration_test.go

```diff
@@ -88,13 +76,6 @@ var _ = Describe("ship init replicated.app/...", func() {
 					// read the test metadata
 					testMetadata = readMetadata(testPath)
 
-					if vendorToken == "" {
```

Comment:
why did we remove this logic?

Reply:
without this things fail very oddly when you forget to set the token, e.g. locally

Reply:
Because we're no longer creating releases
For now
While the vendor api is unstable